myia/prim/composite.py

```diff
@@ -0,0 +1,292 @@
+"""Implementations of primitives as graphs."""
```

Comment:
This should probably not be under `prim/`.

Reply:
Moved to `myia/`